README.md

```diff
@@ -74,7 +74,18 @@ An [AUR package](https://aur.archlinux.org/packages/galaxybudsclient-bin/) for A
 ```
 yay -S galaxybudsclient-bin
 ```
+### Ubuntu installation
```

Comment:
This is not Ubuntu specific

Reply:
yeah sorry about that maybe i could have used desktop entry creation for this